modular_sand/code/modules/mob/dead/new_player/sprite_accessories/body_markings.dm

```diff
@@ -3,3 +3,9 @@
 	icon = 'modular_sand/icons/mob/skyrat_markings.dmi'
 	icon_state = "colorbelly"
 	covered_limbs = list("Chest" = MATRIX_GREEN)
+
+/datum/sprite_accessory/mam_body_markings/floof
+	name = "Belly Fur (Floof)"
+	icon = 'modular_sand/icons/mob/skyrat_markings.dmi'
+	icon_state = "floof"
+	covered_limbs = list("Chest" = MATRIX_RED)
```

Comment:
Please move this to modular_splurt\code\modules\mob\dead\new_player\sprite_accesories\body_markings.dm

And the new marking to modular_splurt/icons/mob/mam_markings.dmi